### DISCUSSION TITLE
Entrar de nuevo a Microsoft Teams

### DISCUSSION THREAD

**Discussion by GerardoCabreraH** [LaunchX-InnovaccionVirtual/CursoIntroPython#47] [Q&A]
Buenas tardes, mi nombre es Gerardo Cabrera y estoy en el programa de Innovacción Virtual, lo que pasa es que tuve que cerrar sesión por error y ya no puedo entrar a Teams, como se va a hacer para que pueda entrar de nuevo la sesión. Mi correo de acceso es este: launchx10957@innovaccion.mx

**Comment by PlayerOneV** [comment]
Si tienes la contraseña que te enviaron por correo junto a tu correo de innovaccion con eso deberías poder iniciar sesión y ya después entrar a la sección de teams.

**Reply by pablo141960** [reply]
pero la contraseña caduca la primera vez que abres el correo, si no la recuerda deberia mandar mensaje al instagram de innovaccion virtual

**Reply by GerardoCabreraH** [reply]
Ya me lo resolvieron el contacto de innovacion, muchas gracias.

**Reply by HectorMiguel71** [reply]
@GerardoCabreraH  ¿Cómo te resolvieron? ¿Serías amable de comentarme por favor?
